test_scripts/Polices/App_Permissions/170_ATF_HP_Order_Of_Request_Response_Notification_On_Registering_App.lua

```diff
@@ -39,6 +39,7 @@ local testCasesForPolicyTableSnapshot = require ('user_modules/shared_testcases/
 
 --[[ General Precondition before ATF start ]]
 commonSteps:DeleteLogsFileAndPolicyTable()
+commonFunctions:SDLForceStop()
```

Comment:
@VVeremjova please clarify with @Itileda usage of this function because in all scripts it is removed from postconditions.